### DISCUSSION TITLE
Fabric version?

### DISCUSSION THREAD

**Discussion by LupusHacker** [djpadbit/Sound-Physics#87] [General]
Is this mod still being updated? Was wondering if there is a version for Fabric.

**Comment by djpadbit** [comment]
The developpement of this mod is essentially on pause. I don't play the newer versions so I don't really find a use for updating it myself. For that matter I don't play minecraft as much I used to, just from time to time and in 1.7.10 or 1.12.2. As for Fabric, i don't really want to switch modding APIs as it's kind of a pain.

But some people already ported the mod to fabric as [Sound-Physics-Remastered](https://github.com/henkelmax/sound-physics-remastered) or [Resounding](https://github.com/thedocruby/resounding).

When I play on newer versions though, I personnally prefer to use [Dynamic Surroundings](https://www.curseforge.com/minecraft/mc-mods/dynamic-surroundings) as it incorporated Sound-Physics's features in the new versions, but it hasn't been updated past 1.16.5 and is for Forge.
